### ISSUE TITLE
Set-Service examples incorrectly claim that Get-WmiObject is needed to determine a service's startup type

### ISSUE THREAD

**Issue by mklement0** [PowerShell/PowerShell-Docs#1889]
https://github.com/PowerShell/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Management/Set-Service.md claims with respect to its example commands:

> These commands use the Get-WmiObject cmdlet to get the Win32_Service object for the service, because the ServiceController object that Get-Service returns does not include the start mode.

However, what `Get-Service` returns - at least as of Windows PowerShell v5.1 / PowerShell Core v6.0.0-rc - _does_ have a `.StartType` property - it's just that the default formatting information does not show it (consider adding it).

```powershell
# The property is clearly there.
> (Get-Service wuauserv).StartType
Manual

# However, it isn't displayed by default.
> Get-Service wuauserv | Format-List
# ... no StartType 

# You must ask for it by name or use *
> Get-Service wuauserv | Format-List *
# ...
StartType           : Manual
# ...
```

Therefore, the example should be revised to use `Get-Service` instead.

Version(s) of document impacted
------------------------------
- [x] Impacts 6 document
- [x] Impacts 5.1 document
- [x] Impacts 5.0 document
- [ ] Impacts 4.0 document
- [ ] Impacts 3.0 document

See below re what versions may or may not be impacted.

**Comment by ScriptAutomate** [comment]
It will be good to do a test on earlier PowerShell versions as I remember this was at least an issue through PowerShell 3 or PowerShell 4. I'm not sure which version of PowerShell modified Get-Service to include it as a property?

**Comment by mklement0** [comment]
@ScriptAutomate:

Good point - you're correct.

https://msdn.microsoft.com/en-us/library/system.serviceprocess.servicecontroller.starttype(v=vs.110).aspx states that the property was added in v4.6.1 of the .NET Framework.

I'm not sure exactly what PowerShell version comes with what .NET Framework version, so for now I'll _assume_ that PSv5+ has access to the property and I've updated the original post accordingly

**Comment by sdwheeler** [comment]
I fixed this for v6 in PR #2556. Need to test this on downlevel versions.
